README.md

```diff
@@ -40,6 +38,29 @@ provisioning:
 ... and that you have a pre-configured `jessie` container that works, you should be able to get
 *something* out of `nomad` commands made in the same folder.
 
+You can also choose to define a container that will be created by pulling directly one of the image
+hosted on https://images.linuxcontainers.org/: this is the "pull" mode:
+
+```
+name: myproject
+image: ubuntu/xenial/amd64
```

Comment:
I'm trying it right now and experience the same issue. What about making `simplestreams` the default protocol? According to my tests here, that's what `lxc` seems to use anyway. If I do `lxc image copy images:debian/stretch local: --copy-aliases`, `lxc image info debian/stretch` tells me that its protocol is `simplestreams`. We might as well mimick `lxc`...